dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/newnonfinancials/FinancialShare.kt

```diff
@@ -0,0 +1,12 @@
+package org.dataland.datalandbackend.model.eutaxonomy.nonfinancials
+
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * This class describes a the relative and absolute share on a different financial asset
+ */
+data class FinancialShare(
```

Comment:
Beim Lesen fand ich den Inhalt der Klasse nicht im Namen reflektiert. Vll etwas in die Richtung absoulteAndrelativeShare?
Dementsprechend auch die beiden Variablen in percantageShare und absoulteShare umbennen?
Was meint ihr?

Reply:
Ja stimme zu.  Wie fändest du das:

`data class RelativeAndAbsoluteFinancialShare(
    val relativeShare: BigDecimal?,
    val absoluteShare: AmountWithCurrency?,
)`

Reply:
Finde ich gut. Nur bei relativeShare sollten wir aufpassen, dass noch deutlich ist, dass ein Prozentwert gemeint ist. Ein relativer Anteil kann ja auch als absoluter Wert ausgedrückt werden.

Reply:
Ja guter Punkt.  Dann vllt:

`data class RelativeAndAbsoluteFinancialShare(
    val relativeShareInPercent: BigDecimal?,
    val absoluteShare: AmountWithCurrency?,
)`

Reply:
Das finde ich gut.

Reply:
Pushed. Schau es dir mal an und sag Bescheid wenn du noch einen Verbesserungsvorschlag hast.

Reply:
Passt, Danke Dir